inheritance/src/main/java/inheritance/Review.java

```diff
@@ -0,0 +1,29 @@
+package inheritance;
+
+public class Review {
+    public String getBody() {
+        return body;
+    }
+
+    public String getAuthor() {
+        return author;
+    }
+
+    public int getNumOfStars() {
+        return numOfStars;
+    }
+
+    private String body;
```

Comment:
It's common convention to list your instance variable first, before anything else in the class. That way future Devon and other developers can easily see what properties the class has instead of having to scroll around to find them